docs/advanced/debug.md

```diff
@@ -3,67 +3,88 @@ title: Enable Kubectl logs/exec to debug pods on the edge
 sidebar_position: 3
 ---
 
-## Prepare certs
+> Note for Helm deployments:
+> - Stream certificates are generated automatically and the CloudStream feature is enabled by default. Therefore, Steps 1-3 can be skipped unless customization is needed.
+> - Step 4 could be finished by iptablesmanager component by default, so manual operations are not needed. Refer to the [cloudcore helm values](https://github.com/kubeedge/kubeedge/blob/master/manifests/charts/cloudcore/values.yaml#L67).
+> - Operations in Steps 5-6 related to CloudCore can also be skipped.
```

Comment:
```suggestion
> - If CloudCore is deploy in container (by default), operations in Steps 5-6 can also be skipped.
```

Overall looks good. : ) just a small suggestion, because there are still some users deploy CloudCore using binary.

Reply:
this is good catch, thanks! fixed in https://github.com/kubeedge/website/pull/661/commits/deb31fb3e98afd185f2e3eca06ef1b4f6232cacf